core/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestClusterAllocationExplainAction.java

```diff
@@ -78,6 +78,7 @@ public RestChannelConsumer prepareRequest(final RestRequest request, final NodeC
                     client.admin().cluster().allocationExplain(req, new RestBuilderListener<ClusterAllocationExplainResponse>(channel) {
                 @Override
                 public RestResponse buildResponse(ClusterAllocationExplainResponse response, XContentBuilder builder) throws Exception {
+                    builder.humanReadable(true);
```

Comment:
What I was advocating for was to enable the human flag by default, but if someone sets human = false to respect that as well...